backend/app/views/spree/layouts/admin.html.erb

```diff
@@ -4,30 +4,43 @@
     <%= render 'spree/admin/shared/head' %>
   </head>
 
-  <body class="admin">
+  <body class="admin <%= "new-layout" if @admin_layout %>">
```

Comment:
This is insisting on adding the class name `new-layout` if @admin_layout is set at all. Is admin_layout a boolean, or is the name of a (I think) class.  Should this be `<%= @admin_layout.presence %>` instead? And should the class name `new-layout` be more descriptive, like `full-width-layout` or something?